src/vendor/github.com/xelabs/go-mysqlstack/driver/server.go

```diff
@@ -123,11 +122,17 @@ func (l *Listener) parserComStatementExecute(data []byte, session *Session) (*St
 	if err != nil {
 		return nil, err
 	}
-	protoStmt, err := proto.UnPackStatementExecute(data[1:], stmt.ParamCount, sqltypes.ParseMySQLValues)
+
+	protoStmt := &proto.Statement{
+		ID:         stmt.ID,
+		ParamCount: stmt.ParamCount,
+		ParamsType: stmt.ParamsType,
+		BindVars:   stmt.BindVars,
+	}
+	_, err = proto.UnPackStatementExecute(data[1:], protoStmt, sqltypes.ParseMySQLValues)
```

Comment:
check in oneline is better :if _, err = ... err!= nil

Reply:
Yeah.

Reply:
Yeah,